tests/registry/ha+tls+external-db_redis/values.yaml

```diff
@@ -0,0 +1,62 @@
+expose:
+  tls:
+    certSource: secret
+    secret:
+      secretName: core-tls
+      notarySecretName: notary-tls
+  ingress:
+    hosts:
+      core: "${NAMESPACE}.${INGRESS_IP}.nip.io"
+      notary: "notary.${NAMESPACE}.${INGRESS_IP}.nip.io"
+    annotations:
+      cert-manager.io/cluster-issuer: letsencrypt
+externalURL: "https://${NAMESPACE}.${INGRESS_IP}.nip.io"
+harborAdminPassword: "Harbor12345"
```

Comment:
this is no longer needed?

Reply:
this is needed until the tests supports running with a password different than the default `Harbor12345`.

Reply:
I see